logger.ts

```diff
@@ -18,11 +18,11 @@ const consoleFormat = winston.format.combine(
 const logger = winston.createLogger({
     format: errorFormat,
     transports:[
-        // writes all logs with level 'error' to the error.log
+        // writes all logs with level 'warn' or higher ('error' too) to the error.log
         new winston.transports.File({
             filename: 'logs/error.log',
-            level: 'error',
-            maxsize: 5242880, // 5MB
+            level: 'warn',
```

Comment:
What is the semantic difference between `error` and `warn` in our case? Because in either case it returns a Webdav-Error, which is displayed as such in the client.

Reply:
logger.warn and logger.error will both be written to the errors.log in this case
I think this is better, as warns are also interesting to inspect (for example if we want to see how often it is attempted to use forbidden chars or so)